Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/API/wmb.yaml

```diff
@@ -0,0 +1,96 @@
+imports:
+  Text: Data.Text
+  APISuccess: Kernel.Types.APISuccess
+  LatLong: Kernel.External.Maps.Types
+  ServiceTierType: Domain.Types.Common
+
+module: WMB
+
+types:
+  RouteInfo:
+    code: Text
+    shortName: Text
+    longName: Text
+    startPoint: LatLong
+    endPoint: LatLong
+
+  AvailableRoutesList:
+    routeInfo: RouteInfo
+    source: StopInfo
+    destination: StopInfo
+    vehicleDetails: VehicleDetails
+
+  StopInfo:
+    name: Text
+    code: Text
+    lat: Maybe Double
+    long: Maybe Double
+
+  VehicleDetails:
+    number: Text
+    tyep: ServiceTierType
```

Comment:
type spelling bro :'(

Reply:
I think it will conflict with type keyword.